src/algebraic_topology/simplex_category.lean

```diff
@@ -759,6 +759,15 @@ begin
   rw [h, eq_id_of_mono θ', category.id_comp],
 end
 
+lemma len_eq_add_succ_of_mono {Δ' Δ : simplex_category} (i : Δ' ⟶ Δ) [hi : mono i]
+  (hi' : ¬ Δ = Δ') : ∃ (k : ℕ), Δ.len = Δ'.len + (k + 1) :=
+begin
```

Comment:
It seems strange that this is not stated as `Δ ≠ Δ'` and `Δ.len < Δ'.len`.

Reply:
Thanks for the suggestions!